contracts/fate.md

```diff
@@ -0,0 +1,808 @@
+# FATE
+The Fast æternity Transaction Engine
+
+## Design
+
+The high level machine (or the Fast æternity Transaction Engine) has
```

Comment:
```suggestion
The high level machine (or the Fast Æternity Transaction Engine) has
```

Reply:
According to our "PR department" we write aeternity in lower case.

Reply:
Not sure how to get the capital A in though ;)

Reply:
Well, logic and PR, don't try to mix those :-)